x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -113,6 +123,16 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
       ];
     }, [os]);
 
+    // use a memoized callback to
+    // limit list iteration for re-renders
+    const operatorOptions = useMemo<Array<EuiSuperSelectOption<string>>>(() => {
+      return (Object.keys(OperatorEntryField) as [ConditionEntry['operator']]).map((value) => ({
```

Comment:
Nit: It looks like that type casting should be `as OperatorEntryField[]` instead of `as [ConditionEntry['operator']]`. 
Although `[ConditionEntry['operator']]` also works, it represents a tuple of one element, which is not accurate.